addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -906,6 +927,14 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         list.dataset.addItemTitle = `Add new ${optionText}`;
         list.dataset.renderListItems = '';
 
+        list.dataset.hasDefault = type === 'one2many' ? 'multiple' : 'unique';
+        const defSelect = type === 'many2one' ? 'selected' : 'checked';
+        const defaults = [...this.$target[0].querySelectorAll(`[${defSelect}="${defSelect}"]`)].map(el => {
```

Comment:
would not it be enough to do:
```suggestion
        const defaults = [...this.$target[0].querySelectorAll('[checked], [selected]')].map(el => {
```
?